### PR TITLE
Doc fixes, part 2

### PR DESCRIPTION
**Pull request by narimiran**
Notes for the reviewer:
- this is a continuation of https://github.com/status-im/nimbus-eth2/pull/4789
- there are two commits, to make it a bit more bearable to review:
  - the majority of the work (and the important stuff to review) is in the first commit ([part 2 of the initial doc fixes](https://github.com/status-im/nimbus-eth2/commit/758e3ed44bfe4c32bad45e59394a39a45920d582))
  - the second commit contains only newline changes
- this is in preparation for the work that needs to be done in https://github.com/status-im/nimbus-eth2/pull/4794

### CODE REVIEW

File: docs/the_nimbus_book/src/audit.md [hunk 1]
```diff
@@ -2,23 +2,28 @@
...
 
-After thoroughly vetting and weighing the submitted proposals, 3 security vendors  were chosen to review the codebase for a timeline of approximately [3 months](https://notes.status.im/7D73zDPyQxOUWw4ejEn6oQ?view#).
+After thoroughly vetting and weighing the submitted proposals, three security vendors  were chosen to review the codebase for a timeline of approximately [three months](https://notes.status.im/7D73zDPyQxOUWw4ejEn6oQ?view#).
 
 The kickoff announcement can be read [here](https://our.status.im/nimbus-beacon-chain-assessment-kickoff/).
 
-We separated the codebase into sub-topics with various tasks. These tasks were then broken up and assigned to the vendor(s) with the required expertise.
+We separated the codebase into sub-topics with various tasks.
+These tasks were then broken up and assigned to the vendor(s) with the required expertise.
```
Comment (arnetheduck) [inline docs/the_nimbus_book/src/audit.md:22]: Eh, all these newlines creates a lot of paragraphs in the rendered output - paragraphs organize / group sentences, so as to make the text easier to follow (ie to separate ideas) and with these changes, this organisation is now lost, to detriment for the reader that now has a more difficult text to navigate - this is a very simple example, but the two sentences about tasks are inherently related, they should not live in separate paragraphs.

Comment (narimiran) [reply docs/the_nimbus_book/src/audit.md:22]: > Eh, all these newlines creates a lot of paragraphs in the rendered output

They don't.

_Single_ newlines in markdown are never rendered as new paragraphs; _double_ newlines are.

The only exception that I know are Github comments like this one, where a single newline is a newline (still not a new paragraph), but all the regular Markdown implementations treat single newlines as whitespace.

I've checked the output of mkdocs's htmls before I decided to make this change throughout the book, and everything was rendered correctly.

Comment (narimiran) [reply docs/the_nimbus_book/src/audit.md:22]: As an example/proof:

There are some .md documents in this book which have newlines inside of the sentences, so the lines are below 80-ish characters wide.
Those (parts of) sentences are not rendered as separate paragraphs, but as regular sentences, like if they were written without any newlines.

Comment (arnetheduck) [reply docs/the_nimbus_book/src/audit.md:22]: oops, you're right, my screen showed an empty line in between the two lines but it was the diff viewer adding it, not an actual newline.

File: docs/the_nimbus_book/src/beacon-node-systemd.md [hunk 2]
```diff
@@ -2,17 +2,20 @@
...
 This page will take you through how to set up a `systemd` service for your beacon node.
 
-`systemd` is used in order to have a command or program run when your device boots (i.e. add it as a service). Once this is done, you can start/stop enable/disable from the linux prompt.
+`systemd` is used in order to have a command or a program run when your device boots (i.e. add it as a service).
+Once this is done, you can start/stop enable/disable from the linux prompt.
 
 !!! abstract "`systemd`"
-    [`systemd`](https://systemd.io/) is a service manager designed specifically for Linux - it cannot be used on Windows / Mac. You can find out more about `systemd` [here](https://fedoramagazine.org/what-is-an-init-system/)
+    [`systemd`](https://systemd.io/) is a service manager designed specifically for Linux - it cannot be used on Windows / Mac.
+    You can find out more about `systemd` [here](https://fedoramagazine.org/what-is-an-init-system/)
```
Comment (zah) [inline docs/the_nimbus_book/src/beacon-node-systemd.md:10]: This sentence is missing a full stop, while other similar info and warning panels seem to have full stops.
That's another thing to normalize in the guide.

**Review by zah — APPROVED** [review]

**Comment by zah** [comment]
I can see a lot of spelling fixes here. Perhaps we can catch them earlier in the authoring process by incorporating a spell checker within the publishing procedure.

**Pull request merged**